app/utils/auth.py

```
@@ -1,97 +1,121 @@
+		
+		mapped_fields = {
+			cls._clerk_payload_mapping[clerk_key]: payload.get(clerk_key)
+			for clerk_key in cls._clerk_payload_mapping
+		}
+		
+		return missing_payload_fields, cls(**mapped_fields)
+
+
+def get_current_user(
+		request_from_context: Request,
+		auth_header: HTTPAuthorizationCredentials = Depends(http_bearer_security_schema),
```

Ruff (0.11.9)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)